rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -23,15 +23,17 @@ import cdm.product.collateral.*
 
 import cdm.mapping.config.*
 
-type TransferableProduct:  <"A TransferableProduct is a type of Product which can be used in a SettlementCommitment for a basic cash settled trade of either an Asset with or without the addition of specific EconomicTerms.">
-    asset Asset (1..1)
-    economicTerms EconomicTerms (0..1)
+type ProductBase: <"Serves as an abstract class to specify a product using a productIdentifier.">
+    productTaxonomy ProductTaxonomy (0..*) <"Specifies the product taxonomy, which is composed of a taxonomy value and a taxonomy source.">
```

Comment:
Consider renaming as just `taxonomy`.

Advantage: same attribute name (and function) between `Asset`, `TransferableProduct` and `NonTransferableProduct`, even if they're not exactly the same type (more advanced inheritance could fix that).

Reply:
P3